src/containers/KatathonContainer/KatathonContainer.js

```diff
@@ -1,10 +1,10 @@
-import React, {Component} from 'react';
+import React, { Component } from 'react'
 
-import Katathon from './Katathon';
+import Katathon from '../../views/Katathon/Katathon'
 
 class KatathonData extends Component {
   constructor(props) {
-    super(props);
+    super(props)
```

Comment:
Do you think we can not use  
state = {
   katas:[
            { 
             userName:'something', 
             score: 0
            }
         ]
     }
With React 16 => We can avoid using constructor(I do not know honestly if it is the best practice, though).
And whenever we have methods we can define them without binding but using arrow function. 
Example: 
onChange = e => {
        this.setState({[e.target.name]: e.target.value});
    }

Reply:
I think these data will come in via redux anyway, so I haven't touched it. I agree we can avoid using the constructor, there is nothing wrong with that. I will leave this state here for now but I will go through and change class methods to arrows so that we don't need to bind them.

Reply:
Good)